components/teams/add-team-member-modal.tsx

```diff
@@ -82,26 +119,48 @@ export function AddTeamMembers({
         <DialogHeader className="text-start">
           <DialogTitle>Add Member</DialogTitle>
           <DialogDescription>
-            You can easily add team members.
+            Invite team members via email or share the invite link.
           </DialogDescription>
         </DialogHeader>
         <form onSubmit={handleSubmit}>
-          <Label htmlFor="domain" className="opacity-80">
+          <Label htmlFor="email" className="opacity-80">
             Email
           </Label>
           <Input
             id="email"
             placeholder="team@member.com"
-            className="mb-8 mt-1 w-full"
+            className="mb-4 mt-1 w-full"
             onChange={(e) => setEmail(e.target.value)}
           />
+          <Button type="submit" className="mb-6 w-full" disabled={loading}>
+            {loading ? "Sending invitation..." : "Send Invitation"}
+          </Button>
+        </form>
 
-          <DialogFooter>
-            <Button type="submit" className="h-9 w-full">
-              {loading ? "Sending email..." : "Add member"}
+        <div className="mb-4">
+          <Label className="opacity-80">Or share invite link</Label>
+          <Input
+            value={inviteLink || ""}
+            readOnly
+            className="mt-1 w-full"
+          />
+          <div className="mt-2 flex justify-between">
+            <Button 
+              onClick={handleCopyInviteLink} 
+              disabled={!inviteLink || inviteLinkLoading}
+              className="flex-1 mr-2"
+            >
+              Copy Link
             </Button>
-          </DialogFooter>
-        </form>
+            <Button 
+              onClick={handleResetInviteLink} 
+              disabled={inviteLinkLoading}
+              className="flex-1"
+            >
+              Reset Link
+            </Button>
+          </div>
+        </div>
```

Comment:
please add the invite by link as a separate dialog / modal

Reply:
I made all changes but I did not understand last changes you told me.
"please add the invite by link as a separate dialog / modal".

Reply:
@ShreyasLakhani instead of adding the button with the copy invite link on the add-team-member-modal.tsx, please create a new component only for copying the invite link. and that should be a separate button on the /people page.